### PR TITLE
Cleaning

### PR DESCRIPTION
**Pull request by 0xABADCAFE**
- Issue #15 - changed all default assignments to `ab3:`
- Removal of obsolete AB3D code references (data mostly)

### CODE REVIEW

**Review by mheyer32 — CHANGES_REQUESTED** [review]

File: ab3d2_source/newloadfromdisk.s [hunk 1]
```diff
@@ -177,261 +177,11 @@ TRYTOOPEN:
...
-;				even
-;RoarName:		dc.b	'AB3D2:sounds/bigscream',0
-				even
-
-;-102
-;7c
-
 OBJNAME:		ds.w	80
 
 OBJ_NAMES:
```
[the remark this answers is not in the document]
Comment (0xABADCAFE) [reply ab3d2_source/newloadfromdisk.s:182]: Sure, was playing it safe.

[the remark this answers is not in the document]
Comment (0xABADCAFE) [reply ab3d2_source/newloadfromdisk.s:182]: OBJNAME, at least, appears to be still be used as a temporary buffer for for the object name in LOADOBS, which is still called from the main control loop.

[the remark this answers is not in the document]
Comment (mheyer32) [reply ab3d2_source/newloadfromdisk.s:182]: Yeah, sure, let OBJNAME be :-)

**Comment by mheyer32** [comment]
In general its amazing how much AB3D cruft was left in TKG...

**Comment by 0xABADCAFE** [comment]
I've removed the symbol and the references in the memory release. It wasn't doing anything useful with it anyway.

**Comment by 0xABADCAFE** [comment]
You're not wrong.

**Comment by mheyer32** [comment]
Thanks for your hard work!

**Pull request merged**